integration/sync_test.go

```diff
@@ -289,6 +314,39 @@ docker.io:
 	c.Assert(nManifests, check.Equals, nTags)
 }
 
+func (s *SyncSuite) TestYamlSemVer2Dir(c *check.C) {
+	tmpDir, err := ioutil.TempDir("", "skopeo-sync-test")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(tmpDir)
+	dir1 := path.Join(tmpDir, "dir1")
+
+	yamlConfig := `
+docker.io:
+  images-by-semver-compare:
+    busybox: "~>1.29.1"
+`
+	var nTags = 3
+	c.Assert(nTags, check.Not(check.Equals), 0)
```

Comment:
The tests are now running for quite a long time, and the purpose of these tests is not to test raw registry interoperability; adding three more image copies is not making that better.

Please use a repository on one of the registries created in `SyncSuite`; maybe pull one small image from `docker.io` and use different tags for that, or copy some of the images from `integration/fixtures` (or add a few more like that) to have clearly different images for the different tags.

Reply:
It seems that all the tests (for sync at least) could do with creating a local docker image with different tags and then using that rather than downloading anything.

I can remove my added integration tests for now, rewriting how the sync tests work would be difficult to justify to my boss at the moment.